boskos/resources.json

```diff
@@ -448,5 +448,15 @@
     ],
     "state": "dirty",
     "type": "gci-qa-project"
+  },
+  {
+    "names": [
```

Comment:
Do you plan to move the existing projects to this pool in a separate PR?

Reply:
We need to do things like whitelist gke, etc, I'll have another PR to create the 4 1.8-gpu jobs to use this pool to start with.